.github/workflows/rnmobile-android-runner.yml

```diff
@@ -0,0 +1,32 @@
+name: React Native E2E Tests (Android)
+on: push
+
+jobs:
+  test:
+    runs-on: macos-latest
+
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: Restore npm cache
+      uses: actions/cache@master
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-npm-${{ hashFiles('package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-npm-
+
+    - uses: reactivecircus/android-emulator-runner@v2
+      with:
+        api-level: 28
+        profile: pixel_xl
+        script: |
+          npm ci
+          npm run native test:e2e:android:local
+
+    - uses: actions/upload-artifact@v2
```

Comment:
Could you explain a bit how this works? Where are the artifacts stored? For how long? How can one access it when looking at a PR?
This could be a nice addition to the mobile doc, otherwise just a comment on this PR so people can find this information using git blame :)

Reply:
Found them here at the top right of the [check screen](https://github.com/WordPress/gutenberg/pull/22189/checks)
![image](https://user-images.githubusercontent.com/230230/81817025-61322f80-952c-11ea-8f11-3774c55e0044.png)

Reply:
In case it's useful, there is some more information here about GitHub actions artifacts: https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts.

@ceyhun :wave: :smile: nice work on this! I :heart: seeing these workflows integrated with GitHub!